avro/src/main/java/com/fasterxml/jackson/dataformat/avro/AvroAnnotationIntrospector.java

```diff
@@ -113,6 +124,10 @@ public Object findSerializer(Annotated a) {
         if (a.hasAnnotation(Stringable.class)) {
             return ToStringSerializer.class;
         }
+        AvroEncode ann = _findAnnotation(a, AvroEncode.class);
+        if (ann != null) {
+            return CustomEncodingSerializer.class;
```

Comment:
Ditto here, my reading would suggest instances or classes both should work?
(but I guess if there's problem with one there's with the other too)